rust/src/ketama.rs

```diff
@@ -2,18 +2,20 @@ use md5::{Digest, Md5};
 
 pub struct KetamaPool {
     /// The list of Servers, sorted by their ranking value.
-    ranking: Vec<ServerRank>,
+    pub ranking: Vec<ServerRank>,
 }
 
-struct ServerRank {
-    value: u32,
-    index: u32,
+pub struct ServerRank {
+    pub server_name: String,
+    pub value: u32,
+    pub index: u32,
```

Comment:
none of these things should be `pub`

Reply:
I agree, we added the `pub` when debugging and I forgot to remove them.